recipes/musicbrainzngs/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "musicbrainzngs" %}
+{% set version = "0.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/musicbrainzngs-{{ version }}.tar.gz
+  sha256: ab1c0100fd0b305852e65f2ed4113c6de12e68afd55186987b8ed97e0f98e627
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - python >=2.7
```

Looks like there are some unsupported 3.x versions, and conda-forge doesn't support 2.7 any more, so it's probably better to do:
```suggestion
    - python >=3.4
  run:
    - python >=3.4
```